src/operations.rs

```diff
@@ -1,8 +1,25 @@
-use actix_web::{HttpRequest, HttpResponse};
-use responses::{HealthcheckSuccess, SuccessfulResponse};
+use actix_web::{HttpRequest, HttpMessage, HttpResponse, AsyncResponder, Error};
+use events::{AccountInvitedToOrg};
+use responses::{HealthcheckSuccess, InviteUserSuccess, SuccessfulResponse};
+use serde_json::{from_slice};
+use bytes::Bytes;
+use futures::future::Future;
 
 pub fn health(_req: &HttpRequest) -> HttpResponse {
     HttpResponse::Ok().json(SuccessfulResponse {
         result: HealthcheckSuccess::new(),
     })
 }
+
+pub fn invite_user(req: &HttpRequest) -> Box<Future<Item = HttpResponse, Error = Error>> {
+    req.body()
+        .limit(1024)
+        .from_err()
+        .and_then(|bytes: Bytes| {
+            println!("==== BODY ==== {:?}", body);
+            let event: AccountInvitedToOrg = from_slice(&bytes).expect("Event convert");
+            Ok(HttpResponse::Ok().json(SuccessfulResponse {
+                result: InviteUserSuccess::new(),
+            }).into())
+        }).responder()
+}
```

Comment:
This entire function can be turned into the following:

```rust
pub fn invite_user(req: HttpRequest) -> Box<Future<Item = HttpResponse, Error = Error>> {
    req.json()
        .from_err()
        .and_then(|val: InviteUserToOrgCmd| {
            println!("model: {:?}", val);
            Ok(HttpResponse::Ok().json(val)) // <- send response
        })
        .responder()
}
```

Where `InviteUserToOrgCmd` looks like this:

```rust
#[derive(Debug, Clone, Serialize, Deserialize)]
pub struct InviteUserToOrgCmd {
    email: String,
    membership_role: MembershipRole,
    name: String,
    organisation_id: OrganisationId,
}
```

It returns a 400 Bad Request if the request body isn't correct. Now you don't need the `bytes` dependency (not sure why Actix wants that when a slice should do...)

We need a better story for validation at some point but for now this will do :)